content/en/blog/_posts/2023-11-16-mid-cycle-1.29.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: blog
+title: 'Kubernetes Removals, Deprecations, and Major Changes in Kubernetes 1.29'
+date: 2023-11-16
+slug: kubernetes-1-29-upcoming-changes
+---
+
+**Authors:** Carol Valencia, Kristin Martin, Abigail McCarthy, James Quigley, Hosam Kamel
+
+
+As with every release, Kubernetes v1.29 will introduce some feature deprecations and removals. Our continued ability to produce high-quality releases is a testament to our robust development cycle and healthy community. The following list are of some of the deprecations and removals coming in the Kubernetes 1.29 release. 
```

Comment:
```suggestion
As with every release, Kubernetes v1.29 will introduce feature deprecations and removals. Our continued ability to produce high-quality releases is a testament to our robust development cycle and healthy community. The following are some of the deprecations and removals coming in the Kubernetes 1.29 release. 
```

Reply:
Actually, ”every release” doesn't introduce deprecations. We could reword (I'm also fine if that happens post-merge).